test/library/apiserver/apiserver.go

```diff
@@ -0,0 +1,123 @@
+package apiserver
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+// WaitForAPIServerToStabilizeOnTheSameRevision waits until all API Servers are running at the same revision.
```

Comment:
what does "all API servers" mean?

Reply:
all pods with `apiserver=true` label specified.

Reply:
updated docs.

Reply:
.... given by the namespaced pod client

Reply:
done